src/diffusers/models/t5_film_transformer.py

```diff
@@ -0,0 +1,303 @@
+# Copyright 2022 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import torch
+from torch import nn
+import math
+from ..configuration_utils import ConfigMixin, register_to_config
+from .modeling_utils import ModelMixin
+from .embeddings import get_timestep_embedding
+from .cross_attention import CrossAttention
+
+
+class T5FilmDecoder(ModelMixin, ConfigMixin):
```

Comment:
The whole T5 Decoder model is now here and uses the optimized `diffusers` cross attention